spec/services/folio_client_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe FolioClient do
+  subject(:client) { described_class.new(url:) }
+
+  let(:url) { 'https://example.com' }
```

Comment:
Should a different host be considered, because example.com is in the allowed list?